riff-cli/Makefile

```diff
@@ -21,6 +21,9 @@ test: build
 $(OUTPUT): $(GO_SOURCES) ../vendor
 	go build -ldflags "-X main.version=$(VERSION)" -o $(OUTPUT) ../cli.go
 
+install: build
+	cp $(OUTPUT) $(GOPATH)/bin
```

Comment:
Should use `$GOBIN`

Reply:
I assume you're suggesting changing this to `cp $(OUTPUT) $(GOBIN)` for consistency with `go install`.
Unfortunately i expect that many users will not have GOBIN set, so if we do this, i suggest also adding the following at the top of the Makefile.
```
GOBIN ?= $(shell go env GOPATH)/bin
``` 
I realize that this will break when there are multiple directories in GOPATH, however that feels a lot less likely.

Reply:
We should use `go env GOBIN` then even.

Reply:
@ericbottard afict `go env GOBIN` returns nothing when GOBIN is unset, which is the case for most users. Am i missing something?

Reply:
I think we could add 
`GOBIN ?= $(GOPATH)/bin`

and use $(GOBIN) in the cp command

Reply:
`GOBIN ?= $(shell go env GOPATH)/bin` will work even when GOPATH is not explicitly set, defaulting to ~/go.

Reply:
Nice, fixed in last commit